TileFormats/VectorData/README.md

```diff
@@ -1,19 +1,248 @@
+| `CYLINDER_BATCH_IDS` | `uint16[]` | The `batchId` of the cylinder that can be used to retrieve metadata from the `Batch Table`. | :red_circle: No. |
+| `ELLIPSOIDS` | `float32[]` | The ellipsoids in the tile. The length of the array will be `19 * ELLIPSOIDS_LENGTH`. The first three elements are the radii in the x, y, and z axes. The following 16 elements are the model matrix. | :white_check_mark: Yes, when the global `ELLIPSOIDS_LENGTH` is greater than zero. |
+| `ELLIPSOID_BATCH_IDS` | `uint16[]` | The `batchId` of the ellipsoid that can be used to retrieve metadata from the `Batch Table`. | :red_circle: No. |
+| `SPHERES` | `float32[]` | The spheres in the tile. The length of the array will be `17 * SPHERES_LENGTH`. The first element is the radius. The following 16 elements are the model matrix. | :white_check_mark: Yes, when the global `SPHERES_LENGTH` is greater than zero. |
+| `SPHERE_BATCH_IDS` | `uint16[]` | The `batchId` of the sphere that can be used to retrieve metadata from the `Batch Table`. | :red_circle: No. |
+
+## Batch Table
+
+The _Batch Table_ contains application-specific metadata, indexable by `batchId`, that can be used for declarative styling and application-specific use cases such as populating a UI or issuing a REST API request.
+
+See the [Batch Table](../BatchTable/README.md) reference for more information.
+
+### Polygon Indices
+
+TODO: `uint16` indices?
+
+The indices are a buffer of `uint32` values. The byte length is given by `polygonIndicesByteLength` in the header. Each count in `POLYGON_INDEX_COUNT` represents a contiguous section of the array that represents a triangulated polygon. 
+For example, let the first two polygons have 6 and 12 for their index counts. The first polygon has 6 indices starting at byte offset `0` and ending at byte offset `6 * byteSize - 1`.
+The second polygon has 12 indices starting at byte offset `6 * byteSize` and ending at `6 * byteSize + 12 * byteSize`.
+
+**Figure 2**: Example index buffer.
+
+![](figures/indices.jpg)
+
+The number of indices must be a multiple of three. Each consecutive list of three indices is a triangle that must be ordered counter-clockwise. Each index is from the start of the buffer, **NOT** from the offset of the first position of the polygon.
+
+### Positions
+
+The positions buffer contains up to three sub-buffers for the polygons, polylines, and points.
+
+The bounding volume for the tile must be a tile bounding region containing the west, south, east, north bounds of the tile. The positions are represented by u, v, and height values that are quantized and delta encoded.
+
+| Field | Meaning |
+| --- | --- |
+| u | The horizontal coordinate of the vertex in the tile. When the u value is 0, the vertex is on the western edge of the tile. Then the value is 32767, the vertex is on the eastern edge of the tile. For other values, the vertex's longitude is a linear interpolation between the longitudes of the western and eastern edges of the tile. |
```

Comment:
With just a few wording tweaks and maybe a property tweak or two - couldn't we word this so that it is not geo specific?  For example, I suspect the code doesn't care that something is "west" or "east", it is just "min" and "max."